Rajitha/itemPrice.js

```diff
@@ -0,0 +1,36 @@
+/**********************************************************************
+ * Create a function that takes an array of objects (groceries) which 
+ * calculates the total price and returns it as a number. A grocery 
+ * object has a product, a quantity and a price,
+ ***********************************************************************/
+const getItemPrice = arr => {
+  let iArr = arr;
```

Comment:
iArr is not needed as you dont need to make a copy of it. 

Try not to create unnecessary variables. Most of the time we use variables for..
1. Describe what we're trying to do example - 
//every 3rd index the item is open - ( i === 3)  this might not be obvious that this means open so we will write it like.
const isOpen = i === 3;
2. We need to make a copy of the data so we can make changes to it without changing the original data.
3. Storing data to process other information.